src/workerd/jsg/setup.h

```diff
@@ -345,7 +352,8 @@ class IsolateBase {
 
   explicit IsolateBase(V8System& system,
       v8::Isolate::CreateParams&& createParams,
-      kj::Own<IsolateObserver> observer);
+      kj::Own<IsolateObserver> observer,
+      kj::Maybe<VfsCallback> vfsCallback = kj::none);
```

Comment:
Any place where we need this to be none? if this is for tests maybe we can just create a mock callback that returns none for every path.
```suggestion
      kj::Maybe<VfsCallback> vfsCallback);
```

Reply:
For now this is set as kj::none in order to keep it from failing on the internal repo where this is not yet implemented. Once we have the internal implementation then this can be updated.